### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by isahers1**
- consistency for roles used in cloud docs
- consistent LangSmith callout everywhere
- consistent setup section

### CODE REVIEW

File: .github/workflows/link_check.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ jobs:
         run: |
           if [ "${{ github.event_name }}" == "schedule" ] || [ "${{ github.event_name }}" == "workflow_dispatch" ] || ([ "${{ github.event_name }}" == "push" ] && [ "${{ github.ref }}" == "refs/heads/main" ]); then
             echo "Running link check on all notebooks in examples directory..."
-            poetry run pytest -v --check-links-ignore "https://(api|web)\.smith\.langchain\.com/.*" --check-links-ignore "https://x.com/.*" --check-links examples
+            poetry run pytest -v --check-links-ignore "https://(api|web)\.smith\.langchain\.com/.*" --check-links-ignore "https://x.com/.*" --check-links-ignore "https://docs.smith.langchain.com" --check-links examples
```
Comment (vbarda) [inline .github/workflows/link_check.yml:58]: @isahers1 you can just add docs to regex i think

Comment (vbarda) [reply .github/workflows/link_check.yml:58]: i.e. `(api|web|docs)`

Comment (isahers1) [reply .github/workflows/link_check.yml:58]: ah yeah I'm silly

File: docs/docs/cloud/how-tos/check_thread_status.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Check the Status of your Threads
+
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/check_thread_status.md:2]:
```suggestion
```

File: docs/docs/cloud/how-tos/background_run.md [hunk 3]
```diff
@@ -1,8 +1,11 @@
 # How to kick off background runs
+
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/background_run.md:2]:
```suggestion
```

File: docs/docs/cloud/how-tos/configuration_cloud.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 # How to create agents with configuration
+
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/configuration_cloud.md:2]:
```suggestion
```

File: examples/rag/langgraph_crag_local.ipynb [hunk 5]
```diff
@@ -47,7 +47,9 @@
     "\n",
     "We'll use a vectorstore with [Nomic local embeddings](https://blog.nomic.ai/posts/nomic-embed-text-v1) or, optionally, OpenAI embeddings.\n",
     "\n",
-    "We'll use [LangSmith](https://docs.smith.langchain.com/) for tracing and evaluation."
+    "We'll use [LangSmith](https://docs.smith.langchain.com/) for tracing and evaluation.\n",
```
Comment (vbarda) [inline examples/rag/langgraph_crag_local.ipynb:50]:
```suggestion
```

File: examples/chatbot-simulation-evaluation/agent-simulation-evaluation.ipynb [hunk 6]
```diff
@@ -207,7 +216,7 @@
    "id": "65bc4446-462b-4ee8-b017-2862fbbdfaf5",
    "metadata": {},
    "source": [
-    "**Nodes**\n",
+    "### Define Nodes\n",
```
Comment (vbarda) [inline examples/chatbot-simulation-evaluation/agent-simulation-evaluation.ipynb:219]:
```suggestion
    "### Define nodes\n",
```

File: examples/rag/langgraph_adaptive_rag.ipynb [hunk 7]
```diff
@@ -462,11 +460,11 @@
    "id": "efbbff0e-8843-45bb-b2ff-137bef707ef4",
    "metadata": {},
    "source": [
-    "# Graph \n",
+    "## Construct the Graph \n",
```
Comment (vbarda) [inline examples/rag/langgraph_adaptive_rag.ipynb:463]:
```suggestion
    "## Construct the graph \n",
```

File: examples/rag/langgraph_adaptive_rag.ipynb [hunk 8]
```diff
@@ -58,43 +60,39 @@
...
+    "</div>    "
    ]
   },
   {
    "cell_type": "markdown",
    "id": "9ac1c2cd-81fb-40eb-8ba1-e9197800cba6",
    "metadata": {},
    "source": [
-    "## Index"
+    "## Create Index"
```
Comment (vbarda) [inline examples/rag/langgraph_adaptive_rag.ipynb:95]:
```suggestion
    "## Create index"
```

File: examples/rag/langgraph_adaptive_rag.ipynb [hunk 9]
```diff
@@ -501,7 +499,7 @@
    "id": "7e2d6c0d-42e8-4399-9751-e315be16607a",
    "metadata": {},
    "source": [
-    "## Graph Flow "
+    "### Define graph Flow "
```
Comment (vbarda) [inline examples/rag/langgraph_adaptive_rag.ipynb:502]:
```suggestion
    "### Define graph flow "
```

File: examples/rag/langgraph_adaptive_rag.ipynb [hunk 10]
```diff
@@ -721,7 +719,7 @@
    "id": "3ab01f36-5628-49ab-bfd3-84bb6f1a1b0f",
    "metadata": {},
    "source": [
-    "## Build Graph"
+    "### Compile Graph"
```
Comment (vbarda) [inline examples/rag/langgraph_adaptive_rag.ipynb:722]:
```suggestion
    "### Compile graph"
```

**Pull request merged**